tests/phpunit/CRM/Contact/Form/Task/EmailCommonTest.php

```diff
@@ -101,15 +88,35 @@ public function testPostProcessWithSignature(): void {
       'signature_text' => 'This is a test Signature',
       'signature_html' => '<p>This is a test Signature</p>',
     ]);
+
+    $to = $form_contactIds = $form_toContactEmails = [];
+    for ($i = 0; $i < 27; $i++) {
+      $email = 'spy' . $i . '@secretsquirrels.com';
+      $contactID = $this->individualCreate(['email' => $email]);
+      $form_contactIds[$contactID] = $contactID;
+      $to[] = $contactID . '::' . $email;
+    }
+    $deceasedContactID = $this->individualCreate(['is_deceased' => 1, 'email' => 'dead@example.com']);
+    $to[] = $deceasedContactID . '::' . 'email@example.com';
+    /* @var CRM_Contact_Form_Task_Email $form*/
+    $form = $this->getFormObject('CRM_Contact_Form_Task_Email', [
+      'to' => implode(',', $to),
```

Comment:
This is the key part of the change to this test - we are now passing 'to' as a submitted value rather than setting properties on the form (this is a good change)